fedbiomed/common/secagg_manager.py

```diff
@@ -608,15 +614,20 @@ def add(self, secagg_id: str, parties: List[str], context: Dict[str, int], exper
             experiment_id: ID of the experiment to which this secagg context element is attached
             context: server key part held by this party
         """
+        # Need to convert bytes to a JSON supported format
+        # Derived keys are not `utf-8` though this is the type reported by `json.detect_encoding()`
+        # Don't modify the original `context` in case it is re-used
```

Comment:
A shorter comment?

```suggestion
        # Save key pairs as base64 since it is the format support by JSON
```

Reply:
Thank you @srcansiz ! The correct comment would be:

```
# Save key pairs as `str`` since it is the format support by JSON. Need to convert to `base64` first
```

Pushing it